ros2_easy_test/decorators.py

```diff
@@ -215,6 +216,7 @@ def with_launch_file(  # noqa: C901
         launch_file: Either:
             1) The path to the launch file to start for the test.
             2) The literal launch file (must contain a newline to be detected as such).
+        parameters: The parameters to be set for the node as ``("key", value)`` pairs
```

Comment:
Here, the key is escaped, but below it is not. Should we escape them on the command line?

Reply:
I do not get exactly what you mean with "below". Here it is just the description of the function argument.

Reply:
I meant in the string that is built below, there are not `"`s. But the documentation suggests there are.

Reply:
Okay, I guess it does not really make sense to support keys or values that need escaping here. And if a user really needs it, they can still escape it themselves.

Let's just do this (see below for the discussion):

 ```suggestion
         arguments: The arguments for the launch file, which will be passed as ``key:=value`` pairs
```